Automattic/simplenote-android
# Update Icon Drawables

### Fix
Update the icon toolbar color tint and drawables.  See the screenshots below for illustration.

![update_icon_drawables_light](https://user-images.githubusercontent.com/3827611/65787174-944abe80-e115-11e9-94cb-e460164e145a.png)

![update_icon_drawables_dark](https://user-images.githubusercontent.com/3827611/65786883-e4755100-e114-11e9-88d4-623ade9e4f6c.png)

### Test
1. Open navigation drawer.
2. Notice updated icons appear as shown above.
3. Close navigation drawer.
4. Notice updated menu and search icons appear as shown above.
5. Tap any note in list that has been published.
6. Notice action icons appear as shown above.
7. Tap ***Share*** action in app bar.
8. Notice icons appear as shown above.
9. Tap back arrow in app bar.
10. Open navigation drawer.
11. Tap ***Edit*** button in ***Tags*** group.
12. Notice icons appear as shown above.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
